### DISCUSSION TITLE
Is there some way in CRI-O to enable experimental features?

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#8021] [Q&A]
If I want to try the upcoming features which have been implemented but still under testing.
Is there some flag to get such features to try them?

**Comment by alpana17** [comment]
@saschagrunert Please suggest

**Comment by saschagrunert** [comment]
Yes, you can use the workloads `allowed_annotations` which are providing some experimental features: https://github.com/cri-o/cri-o/blob/main/docs/crio.conf.5.md#crioruntimeworkloads-table
